bridges/relays/client-substrate/Cargo.toml

```
@@ -14,7 +14,7 @@ jsonrpsee-ws-client = "0.3.1"
 log = "0.4.11"
 num-traits = "0.2"
 rand = "0.7"
-tokio = "1.8"
+tokio = "1.17.0"
```

```suggestion
tokio = "1.8"
```

These kind of unnecessary changes should be done in the bridges repo.

---

If we have code here that is not needed, why do not we delete the whole folder?